src/migrations/0001_initial_schema.sql

```diff
@@ -3,4 +3,9 @@ CREATE TABLE IF NOT EXISTS Clock (
     actorId TEXT,
     seq INTEGER,
     PRIMARY KEY (documentId, actorId)
+) WITHOUT ROWID;
+
+CREATE TABLE IF NOT EXISTS KeyValue (
```

Comment:
What problem are we solving here? Creating a table called KeyValue is, uh, pretty hard for a former database guy to swallow.

Reply:
This is not unappreciated! This is in prep for storing a repo id (public/private key pair) and, upon reflection, should probably be BLOB values rather than TEXT. You could argue we should just write that to disk directly, which would not be a bad argument, or that we could have a specific table for this purpose (and probably spend a little time doing broader schema design), also probably not a bad argument!